code/modules/mapping/turfs.dm

```diff
@@ -0,0 +1,9 @@
+/turf/open/floor/material/meat
+	custom_materials = list(/datum/material/meat = MINERAL_MATERIAL_AMOUNT * 0.25)
+
+/turf/closed/wall/material/meat
+	custom_materials = list(/datum/material/meat = MINERAL_MATERIAL_AMOUNT * 2)
+
+/atom
+	var/prevent_rcd_deconstruction = FALSE
```

Comment:
Should be documented.

Reply:
Also you don't seem to set it to true anywhere.

Reply:
It's for mapping, Duboris wants it for maps

Reply:
He's shilling for sprites

Reply:
i'm shilling for you